### ISSUE TITLE
exec.sh should use absolute paths for input files on JES

### ISSUE THREAD

**Issue by kcibul** [broadinstitute/cromwell#1332]
If I run a WDL like the following in JES, the echo'ed path is relative and I would like it to be absolute.  This will allow me to reference it from other locations than the current working directory more easily.

```
task echo {
  File in_file
  command {
    echo ${in_file}
  }
  runtime {
    docker: "ubuntu:14.04"
  }
}

workflow abs {
  call echo 
}
```

with this input json:

`{
  "abs.echo.in_file": "gs://broad-references/hg38/v0/Homo_sapiens_assembly38.dict"
}`

**Comment by kshakir** [comment]
![19grnh](https://cloud.githubusercontent.com/assets/791985/17954234/7ba0b698-6a47-11e6-873c-c0e60ca163e1.jpg)

I'd be game if all `java.nio.Path`s across cromwell, including the engine, all backends, services, etc. were always absolute. A relative path appearing in a web response, a shell script, or even the logs would then be considered a bug.

In JES, there are internal private methods such as `JesAsyncBackendJobExecutionActor.relativeLocalizationPath` that create relative paths, but these relative paths should not be externally visible. `JesAsyncBackendJobExecutionActor.jesInputsFromWdlFiles` should be creating absolute paths. Over in the SFS, the `SharedFileSystemAsyncJobExecutionActor` doesn't create relative paths, but still inconsistently calls `Path.toAbsolutePath` in various places.

More usage of `better.files` instead of `java.nio.Paths.get()` would help us from omitting calls to `Path.toAbsolutePath`, since [better's `.path` member](https://github.com/pathikrit/better-files#java-interoperability) creates absolute `java.nio.Path`s (for [now](https://github.com/pathikrit/better-files/issues/48#issuecomment-157837460)).

**TL;DR This might also be a problem elsewhere, including the SFS backend.**

**Comment by ruchim** [comment]
@kshakir is this ticket done? I thought I saw a PR flash by? :)
